src/test/java/com/wikia/webdriver/testcases/specialpagestests/EditingPreferencesTests.java

```diff
@@ -27,6 +25,8 @@
   private static final String SOURCE = "1";
   private static final String VE = "2";
   private static final String CK = "3";
+  private static final String username = Configuration.getCredentials().emailQaart2;
+  private static final String password = Configuration.getCredentials().emailPasswordQaart2;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00115)